TRX/Tron-overview.md

```diff
@@ -263,8 +267,13 @@ TRON will always support swapping ERC20 TRX to TRON Mainnet TRX.
 - For Exhanges: Please contact TRON to swap your ERC20 TRX to Mainnet TRX
 
 # 13. Super Representatives and Voting
+The Super Representatives(SR) take important roles to build and operate on TRON network such as block generation and transaction packing. They receive some TRX as rewards. Currently, for the 27 SRs, one block is generated every 3 seconds, with each block awarding 32 TRX to the 27 SRs in sequence; for the 127 SR candidates (including the 27 SRs), there would be 16 additional TRX every 3 seconds to be distributed among the 127 SR candidates in proportion to the votes they have received.
```

Comment:
The top 127 SR candidates now receive `32 TRX` distributed proportionally.

Reply:
A recent proposal is here: https://medium.com/tron-foundation/a-proposal-to-increase-the-value-of-the-rewards-for-the-tron-sr-candidates-d65835ac99b3

> Currently, for the 27 SRs, one block is generated every 3 seconds, with each block awarding 32 TRX to the 27 SRs in sequence; for the 127 SR candidates (including the 27 SRs), there would be 16 additional TRX every 3 seconds to be distributed among the 127 SR candidates in proportion to the votes they have received.

Whenever a block is generated every 3 seconds, the top 27 SRs receive `32 TRX` in sequence. Additionally, the top 127 SR candidates get 32 TRX proportionally based on the amount of votes per node.

what do you think? @Kondax